lib/paperclip/storage/azure.rb

```diff
@@ -166,30 +152,26 @@ def azure_interface
       end
 
       def obtain_azure_instance_for(options)
-        if options[:use_development_storage]
```

Comment:
Hm, I was worried about this at first, but I guess this wasn't ever documented. Dropping it with a notein the changelog seems fine.

Reply:
I didn't know this was a thing before removing it here :grimacing: 
It got me thinking that I could add this functionality back in the gem. Changing major version on the release should help set expectations that the API changed a little bit.